test/cql-pytest/test_tools.py

```diff
@@ -144,10 +144,10 @@ def scylla_sstable(table_factory, cql, ks, data_dir):
     with open(schema_file, "w") as f:
         f.write(schema)
 
-    sstables = glob.glob(os.path.join(data_dir, ks, table + '-*', '*-Data.db'))
-
     try:
-        yield (schema_file, sstables)
+        with nodetool.no_autocompaction_context(cql, f"{ks}.{table}"):
```

Comment:
All tables in this test already use `NullCompactionStrategy` AFAIK. Did you see any problems here?
Using `no_autocompaction_context` here is not incorrect so I won't protest.

Reply:
I thought I saw problems, but may I misdiagnozed it (based on the `NullCompactionStrategy` - that I may have proposed myself in the past now that I'm reminded of it)

Reply:
dropped this patch from the series